docs/api/api/check.xsd

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" 
+	   elementFormDefault="qualified">
+  
+  <xs:annotation>
+    <xs:documentation>
+      This schema describes the format of the status check for the openSUSE API.
+    </xs:documentation>
+  </xs:annotation>
+
+  <xs:element name="check">
+      <xs:annotation>
+        <xs:documentation>
+          Top level element for status check data.
+        </xs:documentation>
+      </xs:annotation>
+      <xs:complexType>
+        <xs:sequence>
+          <xs:element ref="url" minOccurs="0"/>
+          <xs:element ref="short_description" minOccurs="0"/>
+          <xs:element ref="state" minOccurs="1"/>
+        </xs:sequence>
```

Comment:
we don't enforce the element order if not needed usually. (you need to use rng spec instead)

Reply:
Okay...